src/service/transaction.proto

```diff
@@ -0,0 +1,48 @@
+syntax = "proto3";
```

Comment:
for some baffling reason this proto fails to compile

===> Writing /Users/andymck/helium/blockchain-core/src/grpc/autogen/server/transaction_pb.erl
===> Error compiling proto file transaction.proto in msg txn_query_req_v1, field key: undefined reference  blockchain_txn